includes/class-wc-countries.php

```diff
@@ -527,6 +527,7 @@ public function get_address_formats() {
 					'NZ'      => "{name}\n{company}\n{address_1}\n{address_2}\n{city} {postcode}\n{country}",
 					'NO'      => "{company}\n{name}\n{address_1}\n{address_2}\n{postcode} {city}\n{country}",
 					'PL'      => "{company}\n{name}\n{address_1}\n{address_2}\n{postcode} {city}\n{country}",
+					'PR'      => "{company}\n{name}\n{address_1} {address_2}\n{state} {postcode}\n{country}",
```

Comment:
I believe that [postal code generally follows 'country'](https://pe.usps.com/text/pub28/28c2_041.htm) in PR address formats.

```suggestion
					'PR'      => "{company}\n{name}\n{address_1} {address_2}\n{state} \n{country} {postcode}",
```

Reply:
Thanks!